sensor/common/sensor/central_communication_impl.go

```diff
@@ -242,6 +242,7 @@ func (s *centralCommunicationImpl) initialSync(stream central.SensorService_Comm
 	centralid.Set(centralHello.GetCentralId())
 	centralCaps := centralHello.GetCapabilities()
 	centralcaps.Set(sliceutils.FromStringSlice[centralsensor.CentralCapability](centralCaps...))
+	log.Infof("SHREWS -- setting caps %v", centralCaps)
```

Comment:
Leftover?

Reply:
well I was trying to incorporate @lvalerom changes from #8988 and I'm trying to track some timing issues.  I will move this to back to draft for now.